feature-libs/product-multi-dimensional/selector/occ/config/default-occ-product-multi-dimensional-selector-config.ts

```diff
@@ -6,13 +6,19 @@
 
 import { OccConfig } from '@spartacus/core';
 
+const elements =
+  'variantOption(code,variantOptionQualifiers(image(url,format))),variantValueCategory(name),parentVariantCategory(hasImage,name)';
+const variantMatrix = `variantMatrix(${elements},elements(${elements},elements(${elements},elements)))`;
 export const defaultOccProductVariantsMultidimensionalConfig: OccConfig = {
   backend: {
     occ: {
       endpoints: {
         product: {
           multi_dimensional:
-            'products/${productCode}?fields=name,purchasable,baseOptions(DEFAULT),baseProduct,multidimensional,variantOptions(DEFAULT),variantMatrix(DEFAULT),categories',
+            'products/${productCode}?fields=multidimensional,categories,' +
+            variantMatrix,
+          multi_dimensional_guard:
```

Comment:
`minor:` I'd suggest giving it a name that describes the response rather than where it's going to be used.
That said, what do you think about `multi_dimensional_availability`?